control/control.go

```diff
@@ -76,7 +76,8 @@ func (s *SourceParagraph) Maintainers() []string {
 // information that will be eventually put lovingly into the .deb file
 // after it's built on a given Arch.
 type BinaryParagraph struct {
-	Paragraph
+	rfc2822.Paragraph
+
 	Architectures []dependency.Arch `control:"Architecture"`
```

Comment:
Hurm, so, yes, but I'm afraid of API breakage. Can we change it, and log a deprecation warning?